wayland.c

```diff
@@ -129,6 +152,15 @@ static const struct wl_pointer_listener pointer_listener = {
 	.axis = noop,
 };
 
+static const struct wl_touch_listener touch_listener = {
+	.down = noop,
```

Comment:
It's possible to get a down event without a motion event if you press and release without moving. The down event has x, y so we should handle it and update our state.

Reply:
that makes sense!